.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Build images from latest main branch
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  create-main-image:
+    name: env
+    runs-on: ubuntu-latest
+    environment: main
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Build new images and push
+        run: |
+          make \
+            build-docker-images \
+            push-docker-images
+        env:
+          # global
+          DOCKER_USER: ${{ secrets.DOCKER_HUB_USER }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_HUB_PASSWORD }}
+          EOS_PRIV_KEY: ${{ secrets.EOS_PRIV }}
+          MAIL_PASSWORD: ${{ secrets.MAIL_SECRET }}
+          NAMESPACE: lifebank
+          STAGE: dev
+          APP_NAME: lifebank
+          INGRESS_HOST: lifebank.io
+          INGRESS_GRAPHQL_HOST: graphql.lifebank.io
+
```

Comment:
I am not sure if you can leave such empty lines in yaml files.

Reply:
I'm sorry, what do you mean?